### ISSUE TITLE
W3Validator Aria-expanded missing in input

### ISSUE THREAD

**Issue by JakKulGH** [snapappointments/bootstrap-select#2462]
![image](https://user-images.githubusercontent.com/44624526/82317975-63363b80-99cf-11ea-80cc-c446e92035dc.png)

Aria-expanded for button works correctly, but the validator requires aria-eexpanded for the inputs as well

**Comment by hett39** [comment]
+1
